src/race/randomNumber.js

```diff
@@ -0,0 +1,5 @@
+import { Random } from "@woowacourse/mission-utils";
+import { RACE_MOVE } from "../constants/race.js";
+
+export const generateRandomNumber = () =>
+  Random.pickNumberInRange(RACE_MOVE.MIN, RACE_MOVE.MAX);
```

Comment:
상수에 의존하지 않게 `range`를 인자로 받을 수 있게 함수를 구현하고 사용하는 곳에서 상수를 주입 받을 수 있게 만들면 어떨까요?

지금의 구현은 `generateRandomNumber` 함수가 `RACE_MOVE`라는 상수와 강하게 결합되어 있어 재사용하기가 어려울 거 같아요.

```suggestion
export const generateRandomNumber = (min, max) =>
  Random.pickNumberInRange(min, max);
  
 // somewhere
 generateRandomNumber(RACE_MOVE.MIN, RACE_MOVE.MAX);
```